src/Draco.Fuzzer/Testing/Generators/RandomTextGenerator.cs

```diff
@@ -0,0 +1,31 @@
+namespace Draco.Fuzzer.Testing.Generators;
+
+internal sealed class RandomTextGenerator : IInputGenerator<string>
+{
+    private readonly Random random;
+    private readonly int maxLength;
+    private readonly string charset;
+
+    public RandomTextGenerator(int maxLength = 5000, string charset = " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~\r\n\t")
+    {
+        this.random = new Random();
+        this.maxLength = maxLength;
+        this.charset = charset;
+    }
+
+    public RandomTextGenerator(int seed, int maxLength = 5000, string charset = " !\"#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[\\]^_`abcdefghijklmnopqrstuvwxyz{|}~\r\n\t")
+    {
+        this.random = new Random(seed);
+        this.maxLength = maxLength;
+        this.charset = charset;
+    }
```

Comment:
- Factor out this charset constant into a field
 - Propagate to a common constructor